src/components/MultiDropdown/MultiDropdown.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import styles from "./MultiDropdown.module.scss";
+
+var classNames = require("classnames");
+
+export type Option = {
+  /** Ключ варианта, используется для отправки на бек/использования в коде */
+  key: string;
+  /** Значение варианта, отображается пользователю */
+  value: string;
+};
+
+/** Пропсы, которые принимает компонент Dropdown */
+export type MultiDropdownProps = {
+  /** Массив возможных вариантов для выбора */
+  options: Option[];
+  /** Текущие выбранные значения поля, может быть пустым */
+  value: Option[];
+  /** Callback, вызываемый при выборе варианта */
+  onChange: (value: Option[]) => void;
+  /** Заблокирован ли дропдаун */
+  disabled?: boolean;
+  /** Преобразовать выбранные значения в строку. Отображается в дропдауне в качестве выбранного значения */
+  pluralizeOptions: (value: Option[]) => string;
+};
+
+const MultiDropdown: React.FC<MultiDropdownProps> = ({
+  options,
+  value,
+  onChange,
+  disabled,
+  pluralizeOptions,
+}: MultiDropdownProps) => {
+  const [isVisible, setIsVisible] = React.useState<boolean>(false);
+
+  const clickVisible = (): void => {
+    setIsVisible(isVisible ? false : true);
```

Comment:
1. можно покороче и покрасивее
```suggestion
    setIsVisible(!isVisible);
```

2. здесь, особенно если будет мемоизация, могут возникнуть приколы с замыканием, так что лучше положиться на текущее значение состояния не из области видимости, а из того, что может прийти в сеттер состояния:
```suggestion
    setIsVisible((prevIsVisible) => !prevIsVisible);
```